thinkhazard/static/less/report.less

```diff
@@ -39,18 +44,29 @@ body {
   z-index: 2;
 }
 
+#legend ul{
+  list-style-type: none; 
+  display: inline-block;
+  margin-right: 10px;
+  margin-bottom: 30px; 
+  font-size: 18px;
```

Comment:
Looks like the default font-size is 18px.
We should probably set it as the default size for the whole document by overriding `@font-size-base` in `common.less`.